bundles/CoreBundle/Resources/translations/de.json

```diff
@@ -865,6 +865,8 @@
     "this_field_is_required": "This field is required",
     "paste_recursive_as_language_variant": "Paste as new language variant (recursive)",
     "paste_no_new_language_error": "All system languages already linked to source element.",
+    "embedded_meta_info": "Eingebettete Meta Info",
```

Comment:
Please remove the german translation out of the PR, only `EN` is managed manually in Git, all the other languages are managed here https://poeditor.com/projects/view?id=38068 after the EN translation is available in the master branch.